src/Sylius/Behat/Page/Shop/Account/DashboardPage.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page\Shop\Account;
+
+use Sylius\Behat\Page\SymfonyPage;
+
+/**
+ * @author Grzegorz Sadowski <grzegorz.sadowksi@lakion.com>
+ */
+class DashboardPage extends SymfonyPage implements DashboardPageInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getRouteName()
```

Comment:
All of public methods defines the current page somehow. It is not an argument IMHO. But still, I don't mind, to have it defined as a first one. I have only say, that we used to. If you think, that keeping it as a first one is better, please change other pages to be consistent.